lib/components/narrative/narrative-itineraries.js

```diff
@@ -148,6 +150,32 @@ class NarrativeItineraries extends Component {
 
     if (!activeSearch) return null
 
+    // Merge duplicate itineraries together and save multiple departure times
+    let nextIndex = 0
+    const mergedItineraries = itineraries.reduce((prev, cur) => {
```

Comment:
Rename these vars. `prev` to `prevItineraries`... `cur` to `currentItinerary` or `currentItin`?

Reply:
I think that would be more confusing than keeping the `reduce` standard. It's not really the previous itineraries, isn't it more the result from the previous iteration of the loop? I tried to think of better names but `prev` and `cur` are quite standard